pytest_testrail/plugin.py

```diff
@@ -222,6 +224,17 @@ def add_results(self, testrun_id):
         self.results.sort(key=itemgetter('status_id'))
         self.results.sort(key=itemgetter('case_id'))
 
+        # Manage case of "blocked" testcases
+        if self.publish_blocked is False:
+            print('[{}] Option "Don\'t publish blocked testcases" activated'.format(TESTRAIL_PREFIX))
+            blocked_tests_list = [
+                test.get('case_id') for test in self.get_tests(testrun_id)
+                if test.get('status_id') == 2
```

Comment:
Blocked and Skipped are the same status ID in testrail ?
Can this be defined in the status dict at the top.

Reply:
2 is the status of "Blocked" in TestRail. "Skipped" status doesn't exist in TestRail. I don't know why mapping between "skipped" in Pytest and "Blocked" has been made in `PYTEST_TO_TESTRAIL_STATUS` but I don't use it in this case.
I am dealing only with TestRail status here.